.eslintrc

```diff
@@ -23,9 +23,6 @@
         "no-dupe-keys": 2,
         "no-dupe-args": 2,
         "no-else-return": 2,
-        "no-empty": 2,
-        "no-empty-character-class": 2,
```

Comment:
`no-empty` and `no-empty-character-class` still exist is eslint 2. What is the reason to remove the config of those rules here?